client/signup/steps/emails/index.jsx

```diff
@@ -2,78 +2,80 @@
  * External dependencies
  */
 import React from 'react';
-import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
-import page from 'page';
 
 /**
  * Internal dependencies
  */
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+import { getSignupDependencyStore } from 'calypso/state/signup/dependency-store/selectors';
+import EmailStepSideBar from 'calypso/components/emails/email-step-side-bar';
+import EmailSignupTitanCard from 'calypso/components/emails/email-signup-titan-card';
 import StepWrapper from 'calypso/signup/step-wrapper';
-import EmailTitanCard from 'calypso/components/emails/emails-titan-card';
-import EmailSideExplainer from 'calypso/components/emails/emails-side-explainer';
 
 /**
  * Style dependencies
  */
 import './style.scss';
-import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
 
 class EmailsStep extends React.Component {
-	constructor( props ) {
-		super( props );
-	}
+	//TODO
+	handleSkip = () => {};
 
-	getSideContent = () => {
+	renderSideContent = () => {
 		return (
 			<div className="emails__email-side-content-container">
 				<div className="emails__email-side-content">
-					<EmailSideExplainer />
+					<EmailStepSideBar />
 				</div>
 			</div>
 		);
 	};
 
 	renderContent() {
 		const content = (
-			<div className="register-email-step">
+			<div className="emails__register-email-step">
 				<CalypsoShoppingCartProvider>
-					<EmailTitanCard
-						siteUrl={ this.props.progress.domains.siteUrl }
-						addButtonContent={ 'Add' }
-						skipButtonContent={ 'Skip' }
+					<EmailSignupTitanCard
+						siteUrl={ this.props.signupDependencies.domainItem.meta }
```

Comment:
You may want (or need) a null-safe check/path here.
```suggestion
						siteUrl={ this.props.signupDependencies.domainItem?.meta }
```